packages/files/file_path.pony

```diff
@@ -54,43 +54,50 @@ class val FilePath
     """
 
   new val create(
-    base: (FilePath | AmbientAuth),
+    base: FileAuth,
```

Comment:
So I believe this was supposed to be like TCPConnectionAuth which means this should take FileAuth | AmbientAuth.

`TCPConnectionAuth` is type `TCPConnectionAuth is (AmbientAuth | NetAuth | TCPAuth | TCPConnectAuth)`

If this takes `FileAuth | AmbientAuth` then a lot of code that worked previously will still work. IE its less of a breaking change and a ton of the example and test changes you had to do don't have to happen.

I think it makes sense to have a `FilePathAuth` that is `type FilePathAuth is (AmbientAuth | FileAuth)` and make this `FilePathAuth` rather than `FileAuth`.

Reply:
I had the definition of the new `create` in the RFC -- the primary reason for the creation of a "files" root authority was avoiding judicious use of `AmbientAuth`.

Reply:
I didnt read the RFC close enough. I saw "like TCPConnection". You can leave as is, but then I'll open an RFC to add AmbientAuth back in.